app/views/comments/_comments.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="panel panel-info">
+  <div class="panel-heading">
+    Comments
+  </div>
+  <ul class="list-group">
+    <% @comments.each do |comment| %>
+        <li class="list-group-item">
+        <%= comment.user.fullname %>
+        <%= comment.text %>
+        <%= comment.created_at.strftime("%e|%m|%y %H:%M") %>
+          <% if current_user == comment.user %>
+            <%= link_to project_task_comment_path(comment.project, comment.task, comment) ,method: :delete, class: 'inverse-link pull-right', data: { confirm: 'Are you sure?' } do %>
+              <span class="glyphicon glyphicon-remove" aria-hidden="true"></span>
+            <% end %>
+            <%= link_to edit_project_task_comment_path(comment.project, comment.task, comment), class: 'inverse-link edit-link pull-right' do %>
+              <span class="glyphicon glyphicon-pencil" aria-hidden="true"></span>
+            <% end %>
+          <% end%>
+        </li>
+    <% end %>
+</ul>
+</div>
+<%= render partial: 'comments/form', locals: {comment: @comment} %>
```

Comment:
the same as [above](https://github.com/dKotetunov/TasksManager/pull/12/files#r29217560)